index.html

```diff
@@ -1148,6 +1152,23 @@ <h2 id="Model">Backbone.Model</h2>
 });
 </pre>
 
+    <p id="Model-preinitialize">
+      <b class="header">preinitialize</b><code>new Model([attributes], [options])</code>
+      <br />
+      For ES6 classes. If you define a <b>preinitialize</b> method, it will be invoked when
+      the Model is first created and before any instantiation logic is run for the Model.
+    </p>
+
+<pre>
+class Library extends Backbone.Model {
+    preinitialize: function() {
+        this.setValue = 1;
```

Comment:
Can we give a better example for this? Property initializers are pretty easy in es6.